pkgs/development/python-modules/svgelements/default.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, anyio
+, numpy
+, pytest-cov
+, pytest-forked
+, pytest-xdist
+, pytestCheckHook
+, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "svgelements";
+  version = "1.9.6";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "meerk40t";
+    repo = "svgelements";
+    rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```

Helps to avoid branch/tag confusion. Primarily an issue when doing mass-updates.